tests/integration/testlib/ovslib.py

```diff
@@ -47,6 +47,13 @@ def set_options(self, options):
     def add_system_port(self, name):
         self._add_port(name)
 
+    # TODO: Implement adding slaves and mode
+    def add_la_port(self, name):
```

Comment:
How about `add_linkaggregation_port`.

And if you define this helper, it makes sense to use all defined properties in the schema (i.e. the mode and slaves). Otherwise, you can just drop this.